modules/ROOT/pages/index.adoc

```diff
@@ -1,39 +1,205 @@
+Mule Runtime (or Mule) is a lightweight integration engine that runs Mule apps.
+Rather than creating multiple point-to-point integrations between systems,
+services, APIs, and devices, Mule apps manage message routing, data mapping,
+orchestration, reliability, security, and scalability between nodes.
 
-Mule is a lightweight integration engine that runs Mule apps. Rather than creating multiple point-to-point integrations between systems, services, APIs, and devices, you use Mule to create apps that manage message routing, data mapping, orchestration, reliability, security, and scalability between nodes.
+== Event-Driven Architecture
 
-The runtime defines the rules for how event data is handled in an application.  Mule apps receive events, then process and route them to other endpoints.
+Mule works by responding to events that are initiated by external or internal
+resources. Mule supports a variety of listeners, as well as internally
+triggered events based on intervals set in a scheduler. This behavior
+reflects Mule's event-driven architecture (EDA).
 
-Developing with Mule You Can...
+* Mule apps accept and process events as messages through Event processors.
+Mule apps can also handle large messages in batches through Batch processing
+components.
+* Within a Mule app, event processing components are arranged into one or
+more container-like components, called <<flows>>.
++
+For more on this topic, see xref:about-mule-event.adoc[Mule Events].
 
-* Deploy or integrate apps or systems on premise or in the Cloud.
-* Employ out-of-the-box connectors to create SaaS integration apps.
-* Build and expose APIs.
-* Consume APIs.
-* Create Web services that orchestrate calls to other services.
-* Create interfaces to expose apps for mobile consumption.
-* Integrate B2B with solutions that are secure, efficient, and quick to build and deploy.
-* Shift apps to the Cloud.
-* Connect B2B e-commerce activities.
+[[flows]]
+== Flows
 
-== About the Non-blocking Execution Engine
+Understanding basic flow architecture is key to understanding Mule.
+Essentially, every Mule flow contains a series of components that can
+receive or process messages.
 
-The Mule execution engine is based on a non-blocking runtime. This is a task-oriented execution model that allows you to take advantage of non-blocking IO calls and to avoid performance problems.
+.Simple Flow in Design Center
+image::flow-example.png[Simple Flow in Design Center]
 
-Flows always function synchronously. If you need to achieve asynchronous patterns, such as Fire-and-Forget, you can use the Async (`<async>`) component.
+At the simplest level, flows are sequences of Event processors. A message
+that enters a flow can pass through a variety of processors. In a typical
+flow, a Mule app receives a message through an Event source (an endpoint, such
```

Comment:
I wouldn't say "endpoint", that would be confusing for Mule 3 users. Just say HTTP listener would be an example.

Reply:
done

Reply:
done